db-helper.js

```diff
@@ -2,19 +2,23 @@ const db = require('./db-config');
 
 module.exports = {
     getUsers,
-    getUserByUsername,
-    addUser
+    // getUserByUsername,
+    addUser,
+    findBy
```

Comment:
this took so long jsut to find out that you had placed the router before the function was being made